.jenkins/pytorch/win-test.sh

```diff
@@ -40,7 +40,7 @@ if [ -n "$CIRCLE_PULL_REQUEST" ]; then
   file_diff_from_base "$DETERMINE_FROM"
 fi
 
-if [[ "${CIRCLE_JOB}" == *11.1* ]]; then
+if [[ "${CIRCLE_JOB}" == *11.2* ]]; then
   export BUILD_SPLIT_CUDA=ON
```

Comment:
The test failure seems legit. I wonder whether the test failure is related to this flag.